ci/runtests.sh

```
@@ -0,0 +1,44 @@
+  setup_root
+  setup_app
+  setup_env
+  if [ x"${onlysetup}" != x"onlysetup" ]; then
+    echo "RUNNING TESTS: onlysetup=<${onlysetup}>"
+    py.test -sv tests
+  else
+    echo "ONLY PREPARE ENV: onlysetup=<${onlysetup}>"
+  fi
+}
+
+main $@
```

Shellcheck (0.11.0)

[error] Double quote array expansions to avoid re-splitting elements.

(SC2068)